src/Symfony/Component/HttpFoundation/Session/Storage/Handler/NativeFileSessionHandler.php

```diff
@@ -48,8 +48,8 @@ public function __construct($savePath = null)
             $baseDir = ltrim(strrchr($savePath, ';'), ';');
         }
 
-        if ($baseDir && !is_dir($baseDir)) {
-            mkdir($baseDir, 0777, true);
+        if ($baseDir && !is_dir($baseDir) && !@mkdir($baseDir, 0777, true) && !is_dir($baseDir)) {
+            throw new \RuntimeException(sprintf('The Session Storage was not able to create a directory: %s', $baseDir));
```

Comment:
Nope, it was no reports about this. Yep, had to support couple cases when settings were misconfigured and these exceptions pointing problematic configurations really fast.

In 99.9% of cases (I guess) exceptions will not make any harm due to default settings.